frontend/packages/metal3-plugin/src/components/host-detail.tsx

```diff
@@ -28,8 +29,10 @@ import {
 } from '@console/shared';
 
 import { ResourceEventStream } from '@console/internal/components/events';
+import { NodeMaintenanceModel } from '@console/kubevirt-plugin/src/models';
+import { connect } from 'react-redux';
```

Comment:
Vendor imports should go before app/pkg imports. These two import categories should be separated by newline.

```
import * as React from 'react';
import * as _ from 'lodash';
import { Row, Col } from 'patternfly-react';
import { connect } from 'react-redux';

import { stuff } from './stuff';
import { moreStuff } from '@console/foobar';
```

Reply:
fixed